src/modules/outputs/builtins/lint-commit-msg/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+# shellcheck shell=bash
+
+function main {
+  local commit_diff
+  local commit_hashes
+  local current_branch
+  local main_branch=__envBranch__
+  local ci="${!__envCI__:-}"
```

Comment:
@kamadorueda Trying to be able to specify the name of the `CI`. As of now, it looks like both github and gitlab support `CI`, I am not sure about others.